apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOverview/EquipmentSection.tsx

```diff
@@ -88,9 +89,10 @@ function ArtSwapCard({ slotKey }: { slotKey: SlotKey }) {
   const { character: { key: characterKey } } = useContext(CharacterContext)
   const { database } = useContext(DatabaseContext)
   const [show, onOpen, onClose] = useBoolState()
+  const { t } = useTranslation("artifact")
   return <CardLight sx={{ height: "100%", width: "100%", minHeight: 300, display: "flex", flexDirection: "column" }}>
     <CardContent>
-      <Typography><SlotNameWithIcon slotKey={slotKey} /></Typography>
+      <Typography sx={{ display: "flex", gap: 1 }}><SlotIcon iconProps={iconInlineProps} slotKey={slotKey} /> {t<string>(`slotName.${slotKey}`)}</Typography>
```

Comment:
Also not center aligned 
![image](https://user-images.githubusercontent.com/36019388/215018011-6d757bc9-a395-4d5b-a08e-9440c38b045b.png)